pkg/modules/authorization.go

```diff
@@ -492,13 +505,25 @@ func getAuthorizationServerDeployment(op utils.OperatorConfig, cr csmv1.Containe
 			YamlString = strings.ReplaceAll(YamlString, AuthTenantServiceImage, component.TenantService)
 			YamlString = strings.ReplaceAll(YamlString, AuthRoleServiceImage, component.RoleService)
 			YamlString = strings.ReplaceAll(YamlString, AuthStorageServiceImage, component.StorageService)
+			YamlString = strings.ReplaceAll(YamlString, AuthVaultAddress, component.VaultAddress)
 			YamlString = strings.ReplaceAll(YamlString, CSMName, cr.Name)
 		}
 
 		// redis component
 		if component.Name == AuthRedisComponent {
 			YamlString = strings.ReplaceAll(YamlString, AuthRedisImage, component.Redis)
 			YamlString = strings.ReplaceAll(YamlString, AuthRedisCommanderImage, component.Commander)
+			YamlString = strings.ReplaceAll(YamlString, AuthRedisName, component.RedisName)
```

Comment:
// AuthRedisSentinel -
	AuthRedisSentinel = "<AUTHORIZATION_REDIS_SENTINEL>" - Just check if this is being used, I am not seeing anywhere this being referenced.

Reply:
This is used in deployment.yaml for Redis Sentinel StatefulSet and Service.